src/main/java/org/opentestsystem/ap/imrt/iis/service/impl/ItemConverterImpl.java

```
@@ -158,6 +161,13 @@ private void populateItemWithContentRequiredData(final Item item, final BaseItem
             baseItem.setAslAttachments(getContentAttachments(baseItem,
                     item.getAsl().getAttachments(),
                     AttachmentFileTypes.ASL));
+
+            final Instant maxUploadDate = baseItem.getAslAttachments().stream()
+                    .map(Attachment::getUploadedDate)
+                    .max(Instant::compareTo)
+                    .orElse(Instant.MIN);
```

Can this ever not be found (Optional.empty())?  If not, suggest throw since that would be Exceptional rather than defaulting to Min date.  If empty is valid suggest using it rather than null for readability/understandability.